SimDataFormats/GeneratorProducts/src/classes_def.xml

```diff
@@ -206,14 +203,19 @@
 	<class name="edm::Wrapper&lt;LHERunInfoProduct&gt;"/>
 	<class name="edm::Wrapper&lt;LHEEventProduct&gt;"/>
 
- <!--needed for backward compatibility between HepMC 2.06.xx and 2.05.yy-->
- <ioread sourceClass = "HepMC::Polarization" version="[1-10]" targetClass="HepMC::Polarization" source = "" target="m_defined">
+<class name="LHEEventProductLite" ClassVersion="-1">
+    <version ClassVersion="-1" checksum="1637646414"/>
```

Comment:
where should I push the modification to?
here below my issue.

pietro



scram project CMSSW CMSSW_8_0_X_2015-10-29-2300
cd CMSSW_8_0_X_2015-10-29-2300/src/
cmsenv
git cms-merge-topic govoni:MCWeightsSlimmerSecondTry
[...modifications & compile tests]
git add SimDataFormats/GeneratorProducts/src/classes_def.xml
git commit -m "changing class version number"
git push
fatal: No configured push destination.
Either specify the URL from the command-line or configure a remote repository using

    git remote add <name> <url>

and then push using the remote name

    git push <name>